proxy/stat/server.py

```diff
@@ -6,7 +6,33 @@
 from common.stat.metric import StatRegistry, StatSummary, StatGauge
 from common.stat.prometheus import PrometheusServer
 from common.utils.process_pool import ProcessPool
-from .api import STATISTIC_ENDPOINT, TxPoolData, TxFailData, TxDoneData
+from .api import (
+    OpResourceNeonBalanceData,
+    OpResourceSolBalanceData,
+    STATISTIC_ENDPOINT,
+    TxPoolData,
+    TxFailData,
+    TxDoneData,
+)
+
+
+class OpResourceStatApi(AppDataApi):
+    name: ClassVar[str] = "ProxyStatistic::OpResource"
+
+    def __init__(self, registry: StatRegistry):
+        super().__init__()
+        self._op_res_neon_balance = StatGauge("op_res_neon_balance", "Operator resource NEON balance", registry=registry)
+        self._op_res_sol_balance = StatGauge("op_res_sol_balance", "Operator resource SOL balance", registry=registry)
+
+    @AppDataApi.method(name="commitOpResourceNeonBalance")
+    def on_op_res_neon_balance(self, data: OpResourceNeonBalanceData) -> None:
+        label = dict(chain_id=data.chain_id, eth_address=data.eth_address)
```

Comment:
The operator should see the name of the token here, not the chain-id.
You have too options:
- have the list of token names in the stat server, and convert chain-id to token-name here
- or pass the token name from the op-resource service

Reply:
Since `evm_cfg` is available in `free_resource()`, I chose the second option.